certification/internal/policy/operator/deployable_by_olm_test.go

```diff
@@ -0,0 +1,78 @@
+package operator
+
+import (
+	// "fmt"
+	"os"
+	"path/filepath"
+
+	fakecranev1 "github.com/google/go-containerregistry/pkg/v1/fake"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/cli"
+	// log "github.com/sirupsen/logrus"
+)
+
+var _ = Describe("DeployableByOLMCheck", func() {
+	var (
+		deployableByOLMCheck DeployableByOlmCheck
+		engine               cli.OpenshiftEngine
+		imageRef             certification.ImageReference
+		err                  error
+	)
+	const (
+		metadataDir        = "metadata"
+		annotationFilename = "annotations.yaml"
+		annotations        = `annotations:
+  operators.operatorframework.io.bundle.package.v1: testPackage
+  operators.operatorframework.io.bundle.channel.default.v1: testChannel
+`
+	)
+
+	BeforeEach(func() {
+		fakeImage := fakecranev1.FakeImage{}
+		imageRef.ImageInfo = &fakeImage
+
+		// set env var for index image
+		os.Setenv("PFLT_INDEXIMAGE", "test_indeximage")
+		os.Setenv("PFLT_ARTIFACTS", metadataDir)
+
+		// mock bundle directory
+		err = os.Mkdir(metadataDir, 0755)
```

Comment:
When I run this locally, I'm getting an error that it already exists. This should use os.MkdirTemp, and clean it up in AfterEach

Reply:
There is an afterEach block that deleted the metadata directory. I'll try to create the resources under /tmp though.